README.md

```diff
@@ -1,75 +1,61 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Module 1 Capstone
```

Comment:
- [**_SUGGESTION_**] This is really not a descriptive name for this project. The name of the project should be a summary of what the project is all about i.e. Todo app. I can see your app is about "Complete Character academy", something in line with this would look nice and professional in your README.